fake_spectra/spectra.py

```diff
@@ -310,10 +319,13 @@ def _save_multihash(self, save_array, grp):
                     gg.create_group(str(key[ii]))
                     gg = gg[str(key[ii])]
             #Delete old dataset if present
+            #I commented this line to save both ion = -1 and ion = 1 in output spectra
+            """
             try:
                 del gg[str(key[-1])]
             except KeyError:
                 pass
+            """
```

Comment:
I don't understand the need for commenting this?

Reply:
I added this long time ago. But I guess it was because this line was deleting  the `ion=-1`  results.  I should check it again.

Reply:
That's strange: this line isn't doing anything with ion

Reply:
I guess so. I reverted this.